src/vs/workbench/contrib/chat/browser/chatMarkdownDecorationsRenderer.ts

```diff
@@ -87,7 +87,9 @@ export class ChatMarkdownDecorationsRenderer {
 			if (part instanceof ChatRequestTextPart) {
 				result += part.text;
 			} else if (part instanceof ChatRequestAgentPart) {
-				result += this.instantiationService.invokeFunction(accessor => agentToMarkdown(part.agent, false, accessor));
+				if (!part.agent.isDefault) {
+					result += this.instantiationService.invokeFunction(accessor => agentToMarkdown(part.agent, false, accessor));
```

Comment:
How do you get here in this case? You don't have a `ChatRequestAgentPart` for the default agent right?

Reply:
You are right. This is a left-over from a previous attempt in which I would "parse" a synthetic `ChatRequestAgentPart` for prompt with commands of one